app/main.py

```diff
@@ -16,4 +16,29 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {"mutable": [
+                    my_favourite_films,
```

Comment:
<img width="247" alt="Screenshot 2023-08-15 at 18 22 20" src="https://github.com/mate-academy/py-mutable-immutable/assets/117740672/42f25fcd-d2b0-4062-b2cf-5f262c5e6046">
Rewrite your code according to my template